apps/frontend/src/components/global/checklist/ChecklistSeverityOverride.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+  <v-card class="text-center px-8 pt-2" height="300px">
+    <v-card-title class="justify-center"
+      >Severity Override Justification</v-card-title
+    >
+    <v-card-subtitle
+      v-if="selectedRule.severityJustification === ''"
+      class="justify-center mt-1"
+    >
+      <strong>
+        Please input a valid severity override justification. (Required)
+      </strong>
+    </v-card-subtitle>
+    <v-card-subtitle v-else class="justify-center mt-1">
+      <strong>Press "OK" to save.</strong>
+    </v-card-subtitle>
+    <v-textarea
+      v-model="selectedRule.severityJustification"
+      class="mt-2"
+      solo
+      outlined
+      dense
+      no-resize
+      height="130px"
+    />
+    <v-btn color="#616161" dark @click="cancelSeverityOverride"> Cancel </v-btn>
+    <v-btn
+      class="ml-4"
+      color="#616161"
+      dark
+      @click="validateSecurityJustification"
+    >
+      Ok
```

Comment:
some random guide website says to try to use better words/verbs than "ok" so maybe we can see how "override" feels here

Reply:
We can come back to this later if there is any issues